servant-docs/test/Servant/DocsSpec.hs

```diff
@@ -21,6 +22,8 @@ import           Control.Monad
 import           Control.Monad.Trans.Writer
                  (Writer, runWriter, tell)
 import           Data.Aeson
+import           Data.Data
```

Comment:
What, why?

Reply:
I accidentally left that in based on the first version of trying to dynamically generate samples. It's a mistake to have it still in the spec for `servant-docs`: I'll pull it out.